source/docs/software/dashboards/shuffleboard/advanced-usage/shuffleboard-tuning-pid.rst

```diff
@@ -58,7 +58,7 @@ The following example demonstrates how to create a button on your dashboard that
   .. code-block:: c++
 
     frc::ShuffleboardTab& tab = frc::Shuffleboard::GetTab("Shooter");
-    nt::NetworkTableEntry shooterEnable = tab.Add("Shooter Enable", false).GetEntry();
+    nt::GenericEntry& shooterEnable = *tab.Add("Shooter Enable", false).GetEntry();
```

Comment:
Doesn't this return a reference rather than a pointer?

Reply:
It doesn't.
```
shuffleboard/SimpleWidget.h
33:  nt::GenericEntry* GetEntry();
42:  nt::GenericEntry* GetEntry(std::string_view typeString);
```